database/migrations/2016_06_29_205018_create_oauth_client_table.php

```diff
@@ -3,7 +3,7 @@
 use Illuminate\Database\Schema\Blueprint;
 use Illuminate\Database\Migrations\Migration;
 
-class CreateOAuthClientTable extends Migration
+class CreateOauthClientTable extends Migration
```

Comment:
oh yea @DFurnes this was a change i had to make in these migrations that northstar-php made. When I was trying to reset my db I was getting errors that the `CreateOAuthClientTable` didn't exist, even though it did. It was because the file is defined as `create_oauth_client_table` and the casing was off on `oauth` being considered one word. Not sure if this is something you have run into before.